youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/pluginconfig/Content.kt

```diff
@@ -0,0 +1,134 @@
+package com.kaltura.playkit.plugins.youbora.pluginconfig
+
+import com.google.gson.annotations.SerializedName
+
+data class Content (
+    @SerializedName(value = "contentBitrate", alternate = ["bitrate"])
+    var contentBitrate: Long? = null,
+
+    /**
+     * See a list of codes in [
+     * http://mapi.youbora.com:8081/cdns](http://mapi.youbora.com:8081/cdns).
+     */
+    @SerializedName(value = "contentCdn", alternate = ["cdn"])
+    var contentCdn: String? = null,
+
+    @SerializedName(value = "contentCdnNode", alternate = ["cdnNode"])
+    var contentCdnNode: String? = null,
+
+    @SerializedName(value = "contentCdnType", alternate = ["cdnType"])
+    var contentCdnType: String? = null,
+
+    @SerializedName(value = "contentChannel", alternate = ["channel"])
+    var contentChannel: String? = null,
+
+    @SerializedName(value = "contentContractedResolution", alternate = ["contractedResolution"])
+    var contentContractedResolution: String? = null,
+
+    @SerializedName(value = "contentCost", alternate = ["cost"])
+    var contentCost: String? = null,
+
+    @SerializedName(value = "contentDrm", alternate = ["drm"])
+    var contentDrm: String? = null,
+
+    @SerializedName(value = "contentDuration", alternate = ["duration"])
+    var contentDuration: Double? = null,
+    var contentEncodingAudioCodec: String? = null,
+    var contentEncodingCodecProfile: String? = null,
+
+    @SerializedName(value = "contentEncodingCodecSettings", alternate = ["encoding"])
+    var contentEncodingCodecSettings: HashMap<String, String>? = null,
+
+    var contentEncodingContainerFormat: String? = null,
+    var contentEncodingVideoCodec: String? = null,
+
+    @SerializedName(value = "contentEpisodeTitle", alternate = ["title2", "episodeTitle"])
+    var contentEpisodeTitle: String? = null,
+
+    @SerializedName(value = "contentFps", alternate = ["fps"])
+    var contentFps: Double? = null,
+
+    @SerializedName(value = "contentGenre", alternate = ["genre"])
+    var contentGenre: String? = null,
+
+    @SerializedName(value = "contentGracenoteId", alternate = ["gracenoteId"])
+    var contentGracenoteId: String? = null,
+
+    @SerializedName(value = "contentId", alternate = ["id"])
+    var contentId: String? = null,
+
+    @SerializedName(value = "contentImdbId", alternate = ["imdbId"])
+    var contentImdbId: String? = null,
+
+    @SerializedName(value = "contentIsLive", alternate = ["isLive"])
+    var contentIsLive: Boolean? = null,
+    var contentIsLiveNoSeek: Boolean? = null,
```

Comment:
Add the following 
const val KEY_CONTENT_IS_LIVE_NO_SEEK = "content.isLive.noSeek"
        const val KEY_CONTENT_IS_LIVE_NO_MONITOR = "content.isLive.noMonitor"